### PR TITLE
[graphql] Give all queries names to improve datadog representation.

### PR DESCRIPTION
**Pull request by alloy**
Related to artsy/metaphysics#1225

These names are what the modern DataDog GraphQL plugin separates queries by.

### CODE REVIEW

**Comment by alloy** [comment]
Whoever merges first!

**Comment by l2succes** [comment]
👍  Nice.

File: src/desktop/apps/artwork/components/doge/query.coffee [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = """
-  query artwork($id: String!) {
+  query ArtworkDogeQuery($id: String!) {
```
Comment (mzikherman) [inline src/desktop/apps/artwork/components/doge/query.coffee]: 🐕

**Comment by alloy** [comment]
@l2succes It _shouldn’t_, but you know how them 👻 operate. I’ll ask slack to pay attention as well

**Comment by mzikherman** [comment]
Amaze! https://github.com/artsy/force/blob/55f6aad57d4a1b7677b358513f8c8cbb4b5d2b93/src/lib/metaphysics.coffee#L7 might be a place to do the trick of wrapping in a `query` any stragglers that don't start with that string (akin to https://github.com/artsy/metaphysics/pull/1251)

**Comment by alloy** [comment]
Oh yeah, we could probably `console.error(...)` from that function, nice 👌 Will PR that tomorrow

**Pull request merged**